docker-compose.yml

```diff
@@ -1,23 +1,37 @@
 version: '3'
 
 services:
-  db:
+  mysql:
     image: mysql:latest
+    hostname: mysql
```

Comment:
```suggestion
```
The hostname is not needed - it is equal to the service name by default

Reply:
yes, i agree. but just mentioned so it would be clear, in future if some one wants to customize it will easy. we should make this parameterized so that it can be picked up from environment variables.